src/Build/Instance/ProjectInstance.cs

```diff
@@ -248,6 +248,11 @@ public ProjectInstance(string projectFile, IDictionary<string, string> globalPro
         {
         }
 
+        public ProjectInstance(string projectFile, IDictionary<string, string> globalProperties, string toolsVersion, string subToolsetVersion, ProjectCollection projectCollection, EvaluationContext context, bool interactive = false)
```

Comment:
`internal`? Or new public interface OK?

Reply:
Looking at this further, none of the public overloads feature an `EvaluationContext` so I think this was right after all.